doc/topics/windows/windows-package-manager.rst

```diff
@@ -995,51 +975,46 @@ Firefox.
 installer (str)
 ---------------
 
-This is the path to the binary (``.exe``, ``.msi``) that will install the
-package. This can be a local path or a URL. If it is a URL or a Salt path
-(``salt://``), the package will be cached locally and then executed. If it is a
-path to a file on disk or a file share, it will be executed directly.
+The path to the binary (``.exe``, ``.msi``) to install a  package.
+This can be a local path or a URL. If it is a URL or a Salt path (``salt://``),
+then  the package is cached locally and then executed.
+If it is a path to a file on disk or a file share, then it is executed directly.
 
 .. note::
-    When storing software in the same location as the winrepo it is usually best
-    practice to place each installer in its own directory rather than in the
-    root of winrepo.
+    When storing software in the same location as the winrepo then
+    - Create a sub folder named after the package.
+    - Store the package definition file named ``init.sls`` and the binary installer in
+    the same sub folder if you are hosting those files on the ``file_roots``.
 
-    Best practice is to create a sub folder named after the package. That folder
-    will contain the software definition file named ``init.sls``. The binary
-    installer should be stored in that directory as well if you're hosting those
-    files on the file_roots.
-
-    ``pkg.refresh_db`` will process all ``.sls`` files in all sub directories
+.. note::
+    ``pkg.refresh_db`` command processes all ``.sls`` files in all sub directories
     in the ``winrepo_dir_ng`` directory.
 
 install_flags (str)
 -------------------
+The flags passed to the installer for silent installation.
 
-This setting contains any flags that need to be passed to the installer to make
-it perform a silent install. These can often be found by adding ``/?`` or ``/h``
-when running the installer from the command-line. A great resource for finding
-these silent install flags is the WPKG project wiki_:
+You can find these flags by adding ``/?`` or ``/h``when running the installer from the command-line.
```

Comment:
**Nitpick:** You need a space after ``/h`` .

Reply:
Updated